### ISSUE TITLE
Doesn't show stats from commits to private repos

### ISSUE THREAD

**Issue by mikepqr** [tadeuzagallo/GithubPulse#29]
Is there some way to add authentication so GithubPulse shows stats from commits to private repositories (which are hidden to unauthenticated users)?

**Comment by tadeuzagallo** [comment]
Yeah, I've thought about that, it actually works on the chrome extension if you're logged on github on chrome.

But as I was thinking I thought if it'd actually be a good thing or not, as the main idea of project is to contribute to the open source community...

Anyway, in order to work on OS X maybe we could add a place on config so the user could set his token, I'll think about that.

**Comment by kevinjalbert** [comment]
+1 for this feature.

**Comment by caarlos0** [comment]
I think it should be for OOS only. What's the point to commit everyday on private repos?

**Comment by rafaell-lycan** [comment]
+1

**Comment by kevinjalbert** [comment]
I would think either add the ability for private repos to be accounted for in the OS X version, or remove it from the chrome extension. Otherwise there is a larger difference between the two version.

Again, I'm for the inclusion of private repos. Could even do a toggle, or preferences to count private repos.

**Comment by tadeuzagallo** [comment]
You're right, I'll look into how to do that on chrome and try to normalize it on the next version!

**Comment by tadeuzagallo** [comment]
Well, I have at least normalized it, private commits are not shown at all.
I'll try to add the option to insert a token to properly load it later, if anybody feels like doing it, be my guest.

**Comment by battaglr** [comment]
I love this because it doesn't show the commits made on private repos. If you —or anybody— ever add that feature, please, do it in a way we can choose not to see that.

_Great app!_

**Comment by wfalkwallace** [comment]
+1 for the ability to show private repos.

**Comment by tadeuzagallo** [comment]
Sorry guys, I have really been full of work, haven't had enough time to build that :disappointed: 

Something that I really would like to implement is #43, and it's pretty straightforward... 

Maybe if some of you guys feel like doing it :smiley:

**Comment by jansauer** [comment]
:+1:  as long as it is an option

**Comment by DarthMike** [comment]
+1 on the feature

**Comment by bbanbury** [comment]
+1 for ability to show private repos.

**Comment by tanepiper** [comment]
+1 for private repos too - or more specifically DON'T remove it from the Chrome version.  I appreciate other's remarks here about it only showing open source, but at the end of the day not all of us have time to contribute to open source every day or even every week - but why should we then be punished for not having a streak when we have a good week at work?

A better option might be to have an option to filter out private repos if you don't want them.

**Comment by rafaell-lycan** [comment]
@tadeuzagallo now you can show the private commits from other repos on Github.

Settings >> Profile >> [Section] Contributions:
[ x ] Include private contributions on my profile

If you agree, just close this issue.

**Comment by z3t0** [comment]
@rafaell-lycan but it doesnt seem to have any effect on GithHub Pulse? Atleast for me Pulse still doesnt see anything private
